test/es-module/test-esm-command.mjs

```diff
@@ -0,0 +1,31 @@
+import '../common/index.mjs';
+import { spawnSync } from 'node:child_process';
+import { strictEqual } from 'node:assert';
+import { fileURLToPath } from 'node:url';
+
+{
+  const child = spawnSync(process.execPath, [
```

Comment:
Can these tests please use the test runner and `spawnPromisified` like all the recent ESM tests?